executor/simple.go

```diff
@@ -949,7 +948,8 @@ func (e *SimpleExec) executeAlterUser(s *ast.AlterUserStmt) error {
 		if err != nil {
 			return err
 		}
-		err = txn.Commit(tikvutil.SetSessionID(context.TODO(), e.ctx.GetSessionVars().ConnectionID))
+		txn.SetOption(kv.SessionID, e.ctx.GetSessionVars().ConnectionID)
+		err = txn.Commit(context.TODO())
```

Comment:
does this make some log using `logutil.Logger(ctx)` miss connection id?
 
for example https://github.com/pingcap/tidb/blob/3ec8c8bae798642baf267601a5864238d3367b9e/store/tikv/2pc.go#L1033